Zilon.Core/Zilon.BotEnvironment/Startup.cs

```diff
@@ -1,19 +1,19 @@
-﻿using LightInject;
+﻿using System;
+
+using Microsoft.Extensions.DependencyInjection;
 
 using Zilon.Emulation.Common;
 
-namespace Zilon.Bot
+namespace Zilon.BotEnvironment
 {
-    class Startup: InitialzationBase
+    class Startup : InitialzationBase
     {
-        public override void ConfigureAux(IServiceFactory serviceFactory)
+        public override void ConfigureAux(IServiceProvider serviceFactory)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5028977)